checks/packaging.go

```diff
@@ -87,7 +101,13 @@ func isPackagingWorkflow(s, fp string, dl checker.DetailLogger) bool {
 		r2 := regexp.MustCompile(`(?s)npm.*publish`)
 
 		if r1.MatchString(s) && r2.MatchString(s) {
-			dl.Info("candidate node publishing workflow using npm: %s", fp)
+			dl.Info3(&checker.LogMessage{
+				Path: fp,
+				Type: checker.FileTypeSource,
+				// Source file must have line number > 0.
+				Offset: 1,
+				Text:   "candidate node publishing workflow using npm",
```

Comment:
is candidate prefix needed ? Same comment for other uses ?

Reply:
if a workflow is found but not run, the users would see:
```
Info: node publishing workflow using npm
Info: workflow not used in runs
Warn: no publishing workflow detected
```

which is confusing. Using the `candidate` keyword gives:
```
Info: candidate node publishing workflow using npm
Info: workflow not used in runs
Warn: no publishing workflow detected
```

Reply:
Resolving, but feel free to disagree and I'll follow-up with a PR to fix it.